examples/docs_snippets/docs_snippets/guides/components/integrations/census-component/6-list-defs.txt

```diff
@@ -0,0 +1,7 @@
+$ dg list defs
+
+Component defs/census_ingest (type dagster_census.CensusComponent):
```

Comment:
is this the actual output you get when you run `pytest <this file> --update-snippets` or was this generated a different way? I'm used to this outputting a more tabular format, e.g. https://sourcegraph.com/github.com/dagster-io/dagster/-/blob/examples/docs_snippets/docs_snippets/guides/components/integrations/dlt-component/9-list-defs.txt